### PR TITLE
chore: release notes for 7.1.0

### PR DESCRIPTION
**Pull request by ecormany**
# Overview

Adds release notes for 7.1.0.

# Test Plan

- Check the markdown preview on GitHub.
- Post-merge, check how these appear in the app and robot update flows.

# Changelog

- Updates to the usual two `release-notes.md` files.
- Added some trailing periods in 7.0.2 notes. 🧐 

# Review requests

Any features that I missed or miscategorized (app vs. API)?

# Risk assessment

Low. No code, but high-visibility text once we reach stable.

### CODE REVIEW

**Review by sfoster1 — APPROVED** [review]
Looks good to me if you're happy with the copy. We might add more stuff but I think we should do that in separate PRs and merge this one so the alphas actually say the right version.

File: app-shell/build/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,23 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
+### New features
+
+- Specify the deck configuration of Flex, including the movable trash bin, waste chute, and staging area slots.
+- Resolve conflicts between the hardware a protocol requires and the current deck configuration as part of run setup.
+- Run protocols that use the Flex 96-Channel Pipette, including partial tip pickup.
+- Choose where to dispense liquid and drop tips held by a pipette when a protocol is canceled.
+
+### Bug Fixes
+
+- Labware Position Check no longer tries to check the same labware in the same position twice, which was leading to errors.
```
Comment (jwwojak) [inline app-shell/build/release-notes.md:22]: All of this LGTM, but I'd like to point out this part of the bug fix note:

> which was leading to errors.

😬   Is it enough to just use the rest of the sentence without this part (the "which was leading to").

**Review by jwwojak — COMMENTED** [review]
LGTM save for one comment on line 22, bug fixes. The "which was leading to . . ." is a wee awkward. Maybe drop everything after the comma an just end the sentence after the explanation.

**Comment by sfoster1** [comment]
also swapped a couple characters in the pr title

**Comment by ecormany** [comment]
> also swapped a couple characters in the pr title

lol thx. merging now, will open new PRs for later alphas if needed.

File: app-shell/build/release-notes.md (hunk 1 above)
Comment (ecormany) [reply app-shell/build/release-notes.md:22]: I think it's good to indicate we fixed the root cause and eliminated the errors. Hopefully reassuring to anyone who encountered them 😁

**Pull request merged**